lib/longtail_platform.c

```diff
@@ -1066,19 +1066,30 @@ int Longtail_GetFilePermissions(const char* path, uint16_t* out_permissions)
     return 0;
 }
 
+#define LONGTAIL_WIN32_PART_SIZE(cur, end) \
```

Comment:
I'm not to fond of this macro - I'd rather have a definition like MAX_CHUNK_SIZE - preferrably a constant rather than a macro and write the terniary expression inline. It is just in two places one for read and one for write.

Reply:
I tried to avoid "chunk" in the name because of the overloaded meaning in the code base, but agreed MAX_CHUNK_SIZE is a more common term for this.